pkg/reconciler/autoscaling/kpa/resources/decider.go

```diff
@@ -73,6 +75,25 @@ func MakeDecider(ctx context.Context, pa *v1alpha1.PodAutoscaler, config *autosc
 			PanicThreshold:      panicThreshold,
 			StableWindow:        resources.StableWindow(pa, config),
 			ServiceName:         svc,
+			InitialScale:        GetInitialScale(config, pa),
 		},
 	}
 }
+
+// GetInitialScale returns the calculated initial scale based on the autoscaler
+// ConfigMap and PA initial scale annotation value.
+// TODO(taragu): This function is exported and will be reused in other packages.
+func GetInitialScale(asConfig *autoscalerconfig.Config, pa *v1alpha1.PodAutoscaler) int32 {
```

Comment:
think this can be private

Reply:
this function will be used in another package in a followup PR

Reply:
What package, asking for a friend? :)

Reply:
@vagababov `pkg/reconciler/autoscaling/kpa/` 😆

Reply:
That I can live with :)